Remotion/Web/Development.WebTesting/WebTestActions/ClickAction.cs

```diff
@@ -43,7 +43,7 @@ protected override void ExecuteInteraction (ElementScope scope)
     {
       ArgumentUtility.CheckNotNull ("scope", scope);
 
-      if (scope.Browser.IsInternetExplorer())
+      if (scope.Browser.IsFirefox() || scope.Browser.IsInternetExplorer())

```

Comment:
Could you link the RM or Firefox issue here and indicate that this is a workaround?